pkg/segment/aggregations/generateevents.go

```diff
@@ -0,0 +1,91 @@
+package aggregations
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/siglens/siglens/pkg/segment/structs"
+	"github.com/siglens/siglens/pkg/segment/utils"
+)
+
+func getFormattedTime(t time.Time) string {
+	return t.Format("Mon Jan 2 15:04:05 2006 -0700")
+}
+
```

Comment:
nit: Move this to `timutils.go` and rename this function to something more descriptive. I guess taking inspiration from Splunk: `formatToHumanReadableTime`. 
And probably also add a comment to this function with the format.

Reply:
addressed in the latest commit.